src/apps/common/submodules/mediaSelect/mediaSelect.js

```diff
@@ -11,6 +11,22 @@ define(function(require) {
 			'common.mediaSelect.render': 'mediaSelectRender'
 		},
 
+		appFlags: {
+			mediaSelect: {
+				allowedFiles: ['audio/mp3', 'video/mp4']
+			}
+		},
+
+		/**
+		 * @param {Object} args
+		 * @param {jQuery} args.container
+		 * @param {String} [args.skin]
+		 * @param {Function} args.callback
+		 * @param {String} [args.label]
+		 * @param {Array} args.options
+		 * @param {Array} [args.allowedFiles]
```

Comment:
[MIME](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types) is commonly used to refer to this sort of types:
```suggestion
		 * @param {Array} args.mimeTypes
```